scripts/telegram_ros_bridge

```diff
@@ -24,6 +26,9 @@ class TelegramROSBridge(object):
         """
         self._caption_as_frame_id = caption_as_frame_id
 
+        # Whitelist is defaults to False to disable it and allow anyone. To enable the whitelist, set it to an (empty) list
+        self._whitelist = rospy.get_param(WHITELIST, False)
```

Comment:
No need to keep this as a member variable, because you update it every time you use it.